From 948c947ba86ddb0782b065a1830c60c334ccb73a Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:40:10 +0200
Subject: [PATCH 5178/5242] MLK-20328-6: ARM64: dts: mx8mq-evk: ak4497: adjust
 the expected format

commit  c667eeda420f5f2e2cfbf4d7a9fa036d31fbdc8a from
https://source.codeaurora.org/external/imx/linux-imx.git

Default value must be prefixed by "0".

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit a38075f068fd07dda55cb5bb5d74450ba29d6483)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8mq-evk-ak4497.dts   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-ak4497.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-ak4497.dts
index 359894e..b761d4c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-ak4497.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-ak4497.dts
@@ -89,7 +89,7 @@
 	assigned-clock-parents = <&clk IMX8MQ_AUDIO_PLL2_OUT>;
 	assigned-clock-rates = <45158400>;
 	fsl,sai-multi-lane;
-	fsl,dataline,dsd = <0xff 0x11>;
+	fsl,dataline,dsd = <0 0xff 0x11>;
 	dmas = <&sdma2 8 26 0>, <&sdma2 9 26 0>;
 	status = "okay";
 };
-- 
1.7.9.5

